pkg/mover/chart.go

```diff
@@ -413,8 +417,13 @@ func (cm *ChartMover) pushRewrittenImages(imageChanges []*internal.ImageChange)
 		if change.ShouldPush() {
 			err := retry.Do(
 				func() error {
-					cm.logger.Printf("Pushing %s...\n", change.RewrittenReference.Name())
-					err := cm.targetContainerRegistry.Push(change.Image, change.RewrittenReference)
+					imageToPush := change.RewrittenReference
+					if change.Tag != "" {
+						imageToPush, _ = name.NewTag(imageToPush.Context().Name(), name.WithDefaultTag(change.Tag))
```

Comment:
`NewTag` returns an error, I do not think we should swallow it here. Even if we think it may not happen.

Reply:
Thanks, that was bad code hygiene on my part. I added the err check and will revert to the old method if it fails.